pydarn/plotting/rtp.py

```diff
@@ -0,0 +1,438 @@
+# Copyright (C) 2019 SuperDARN
+# Author: Marian Schmidt
+# This code is improvement based on rti.py in the DaVitpy library
+# https://github.com/vtsuperdarn/davitpy/blob/master/davitpy
+
+"""
+Range-time Intensity plots
+"""
+import matplotlib.pyplot as plt
+from matplotlib import dates, colors, cm, ticker
+import numpy as np
+from typing import List
+from datetime import datetime, timedelta
+
```

Comment:
PEP8 the import order here and throughout (as before)

Reply:
done